usmqe/api/tendrlapi/cephapi.py

```diff
@@ -19,3 +19,335 @@ def import_cluster(self, nodes, asserts_in=None):
+            "reason": 'Accepted',
+            "status": 202}
+
+        self.print_req_info(response)
+        self.check_response(response, asserts_in)
+        return response.json()
+
+    def get_pool_list(self, cluster, asserts_in=None):
+        """ Get pool list for specific Ceph cluster.
+
+        Name:        "get_pool_list",
+        Method:      "GET",
+        Pattern:     ":cluster_id:/GetPoolList",
+
+        Args:
+            cluster (str): Cluster ID.
+            asserts_in (dict): assert values for this call and this method
+        """
+        pattern = "{}/GetPoolList".format(cluster)
+        response = requests.get(
+            pytest.config.getini("usm_api_url") + pattern,
+            auth=self._auth)
+        self.print_req_info(response)
+        self.check_response(response, asserts_in)
+        return response.json()
+
+    def delete_pool(self, cluster, pool_id, asserts_in=None):
+        """ Delete Ceph pool.
+
+        Name:        "delete_pool",
+        Method:      "DELETE",
+        Pattern:     ":cluster_id:/CephDeletePool",
+
+        Args:
+            cluster (str): Cluster ID
+            pool_id (str): Pool ID
+            asserts_in (dict): assert values for this call and this method
+        """
+        pattern = "{}/CephDeletePool".format(cluster)
+        pool_data = {"Pool.pool_id": pool_id}
+        response = requests.delete(
+            pytest.config.getini("usm_api_url") + pattern,
+            json=pool_data,
+            auth=self._auth)
+        asserts_in = asserts_in or {
+            "cookies": None,
+            "ok": True,
+            "reason": 'Accepted',
+            "status": 202}
+        self.print_req_info(response)
+        self.check_response(response, asserts_in)
+        return response.json()
+
+    def create_rbd(self, cluster, pool_id, name, size, asserts_in=None):
+        """ Create RBD in Ceph pool.
+
+        Name:        "create_rbd",
+        Method:      "POST",
+        Pattern:     ":cluster_id:/CephCreateRbd",
+
+        Args:
+            cluster (str): Cluster ID
+            pool_id (str): Pool ID
+            name (str): RBD name
+            size (int): RBD size
+            asserts_in (dict): assert values for this call and this method
+        """
+        pattern = "{}/CephCreateRbd".format(cluster)
+        pool_data = {"Rbd.pool_id": pool_id,
+                     "Rbd.name": name,
+                     "Rbd.size": size
+                     }
+        response = requests.post(
+            pytest.config.getini("usm_api_url") + pattern,
+            json=pool_data,
+            auth=self._auth)
+        asserts_in = asserts_in or {
+            "cookies": None,
+            "ok": True,
+            "reason": 'Accepted',
+            "status": 202}
+        self.print_req_info(response)
+        self.check_response(response, asserts_in)
+        return response.json()
+
+    def update_rbd(self, cluster, pool_id, name, size, asserts_in=None):
```

Comment:
Ok, I just found it strange that the pattern for API call is named Resize and Function is Update.